rollup.config.js

```diff
@@ -1,13 +1,13 @@
-import { uglify } from "rollup-plugin-uglify";
+import { uglify } from 'rollup-plugin-uglify';
 
 export default {
-  input: "src/index.js",
+  input: 'src/index.js',
   output: {
-    format: "umd",
-    name: "scoped-style",
-    file: "scoped-style.js",
-    dir: "dist/",
-    sourcemap: true
+    format: 'umd',
+    name: 'scopedStyle',
```

Comment:
I couldn't create TypeScript namespace named with a dash (scoped-style) - it is possible when creating a module declaration, but as far as I know, to support UMD it has to be a namespace.

I'm not sure how important it is to have hyphen there (? @sadick254 @halcaponey), let me know.

Reply:
up